index.js

```diff
@@ -156,6 +193,36 @@ const createGlobalDynamodbTable = async function createGlobalDynamodbTable(serve
   }
 }
 
+const deleteGlobalAndBackupTables = async function deleteGlobalAndBackupTables(serverless) {
+  try {
+    serverless.cli.consoleLog(`DeleteGlobalTable: ${chalk.yellow('Starting to delete backup region global tables...')}`)
+    const awsCredentials = serverless.getProvider('aws').getCredentials()
+
+    const globalTablesOptions = serverless.service.custom.globalTables
+    if (!globalTablesOptions || globalTablesOptions.length === 0) {
+      return
+    }
+    await Promise.all(globalTablesOptions.map((option) => {
+      if (option.tableKey) {
+        serverless.cli.consoleLog(`DeleteGlobalTable: ${chalk.yellow('Warning: backup region global tables not removed. ' +
```

Comment:
@johndww why cant we delete the global table if original table is part of the service?